index.js

```diff
@@ -207,11 +207,15 @@ class ExecutorQueue extends Executor {
         }
 
         return new Promise((resolve, reject) => {
-            req(options, (err, response) => {
+            req(options, (err, response, body) => {
```

Comment:
what's in err ?

Reply:
`err` is only when something is wrong with request module.
most response, the err is in the body like this:
```
Error : null
Body : {"statusCode":404,"error":"Not Found","message":"Not Found"}
```